packages/editor/src/client/routes/polls/pollList.tsx

```diff
@@ -5,32 +5,58 @@ import {useTranslation} from 'react-i18next'
 import {Link} from 'react-router-dom'
 import {FlexboxGrid, IconButton, Message, Pagination, Table, toaster} from 'rsuite'
 
-import {FullPoll, usePollsQuery} from '../../api'
+import {Poll, usePollsQuery} from '../../api'
 import {createCheckedPermissionComponent} from '../../atoms/permissionControl'
 import {CreatePollBtn} from '../../atoms/poll/createPollBtn'
 import {DeletePollModal} from '../../atoms/poll/deletePollModal'
 import {PollStateIndication} from '../../atoms/poll/pollStateIndication'
 import {dateTimeLocalString, DEFAULT_MAX_TABLE_PAGES, DEFAULT_TABLE_PAGE_SIZES} from '../../utility'
 
+export function PollOpensAtView({poll}: {poll: Poll}) {
```

Comment:
Moved these out as standlone components so I could reuse them for the poll list in the poll block